cmd/k3sctl/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/datawire/teleproxy/pkg/dtest"
+
+	"github.com/spf13/cobra"
+)
+
+// Version holds the version of the code. This is intended to be overridden at build time.
+var Version = "(unknown version)"
+
+func main() {
+	var k3s = &cobra.Command{
+		Use:           "k3sctl",
+		Short:         "k3sctl",
+		Long:          "k3sctl - manage a local docker registry and kubernetes cluster",
```

Comment:
I think this should be 

```Go
Short: "manage a local Docker registry and Kubernetes cluster",
```

`Short` is for a 1-liner human description of the program, `Long` is for a paragraph description.